tracer/src/Datadog.Trace/Debugger/Snapshots/Redaction.cs

```diff
@@ -78,125 +78,114 @@ internal static class Redaction
 
         private static HashSet<string> _redactKeywords = new()
```

Comment:
Would it be preferable to make this a case-insensitive comparer?
```suggestion
        private static HashSet<string> _redactKeywords = new(StringComparer.OrdinalIgnoreCase)
```

Reply:
In which case, you can use C#12 collection literals! 😄

Reply:
I'm not sure I understand, can you give me a code suggestion, please?

Reply:
I understand now, will correct it next PR I will on `redaction` feature